### PR TITLE
Introducing public documentation of hveto

### PR DESCRIPTION
**Pull request by alurban**
This PR introduces a number of changes for public documentation of `hveto` hosted at https://hveto.readthedocs.io:

* Updated documentation indices and `conf.py` script
* New YAML configuration for readthedocs (`.readthedocs.yml`)
* Updated badges in `README.rst`, including one for the latest version on conda-forge
* Updated `.gitignore`
* Typo fix in `hveto-trace`

Note, I've already imported hveto to readthedocs.org, so the updated documentation will build automatically for every new release (stable) and every commit to `master` (latest), as soon as this PR is merged.

cc @jrsmith02, @duncanmmacleod

### CODE REVIEW

**Comment by alurban** [comment]
@jrsmith02, here's a preview of what the rendered docs pages look like:

Screen Shot 2019-04-22 at 09 43 45

**Comment by jrsmith02** [comment]
This looks really good to me. Love that it has a doi for citing, etc. My only question is how does this page relate to this other page: https://ldas-jobs.ligo.caltech.edu/~duncan.macleod/hveto/latest/ ? Does it overwrite it? Might it be useful to still say how to run a basic version of get help?
Thanks

**Comment by alurban** [comment]
@jrsmith02, responses below:

* The old documentation was built in a slightly hacky way, on a private user account, hidden behind `LIGO.ORG` authentication, and never updated unless someone had the time for it (so never updated).
* That old documentation will still technically exist unless Duncan deletes it. The new docs will be publicly hosted on readthedocs, at the domain http://hveto.readthedocs.io, and automatically updated every time this repository's `master` branch is, or every time there's a release. This allows us much better control and will increase visibility of the documentation pages.
* Anyone who contributes to the project will have a public citable DOI and publicly viewable links to the documentation, which is a plus.
* What's shown above is just a screenshot, and it doesn't show the full landing page, which has information about installation and contributing as well as a table of contents. The top navbar has a drop-down menu that can be used to navigate to extensive documentation about running hveto on the command-line, which I expanded onto its own page from what was there before, and each submodule of hveto, which is automatically built from function docstrings.

tl;dr: no information has been removed, and a lot of it has been significantly expanded.

**Comment by jrsmith02** [comment]
Got it! Thank you sir. Merging.
